src/bokeh/embed/bundle.py

```diff
@@ -236,10 +239,10 @@ def bundle_for_objs_and_resources(objs: Sequence[Model | Document] | None,
 # Private API
 #-----------------------------------------------------------------------------
 
-def _query_extensions(objs: Sequence[Model | Document], query: Callable[[type[Model]], bool]) -> bool:
+def _query_extensions(all_objs: Set[Model], query: Callable[[type[Model]], bool]) -> bool:
```

Comment:
Use `set[T]` instead of `Set[T]` everywhere.

Reply:
Oh, I can never seem to recall which ones to use. Fixes, thanks!

Reply:
Originally only `Set[T]` (and similar, `List[T]`, `Dict[K, V]`) was allowed. Since Python 3.9 (https://peps.python.org/pep-0585/) both are allowed and we prefer `set[T]` over `Set[T]` as it reduces the number of imports.